### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Q-lds**
Fixing the squash merge madness

### CODE REVIEW

File: docs/source/problems.rst [hunk 1]
```diff
@@ -59,6 +59,11 @@ Helper functions
     :show-inheritance:
     :inherited-members:
 
+.. autoclass:: openqaoa.problems.problem.ShortestPath
```
Comment (shahidee44) [inline docs/source/problems.rst:62]: Is this block duplicated?

File: openqaoa/backends/qpus/qaoa_pyquil_qpu.py [hunk 2]
```diff
@@ -130,6 +129,8 @@ def __init__(self,
 
         # check_edge_connectivity(self.prog_exe, device)
 
+        # check_edge_connectivity(self.prog_exe, device)
```
Comment (shahidee44) [inline openqaoa/backends/qpus/qaoa_pyquil_qpu.py:132]: Is this block duplicated?

File: openqaoa/workflows/optimizer.py [hunk 3]
```diff
@@ -178,6 +178,14 @@ def set_circuit_properties(self, **kwargs):
                     "Specified argument is not supported by the circuit")
         self.circuit_properties = CircuitProperties(**kwargs)
 
+        for key, value in kwargs.items():
```
Comment (shahidee44) [inline openqaoa/workflows/optimizer.py:181]: Is this block duplicated?

File: tests/test_optimisers.py [hunk 4]
```diff
@@ -26,6 +26,9 @@
 from openqaoa.optimizers.logger_vqa import Logger
 from openqaoa.qfim import qfim
 
+from openqaoa.optimizers.logger_vqa import Logger
```
Comment (shahidee44) [inline tests/test_optimisers.py:29]: Is this block duplicated?

**Review by shahidee44 — COMMENTED** [review]
I think its mostly just duplicated code

**Comment by vishal-ph** [comment]
Yeah, looks like there are some possible duplications as pointed our by Shahidee.

**Comment by Q-lds** [comment]
@vishal-ph and @shahidee44 this is very possible. Then we can just remove them. I can do it a few hours, if someone can do it right away: go ahead!

**Comment by shahidee44** [comment]
I've push a fix from #74 . If either of you approve we can merge it in and the branch should be good to go.
@vishal-ph @Q-lds

**Comment by vishal-ph** [comment]
thanks @shahidee44, I merged your PR and fixed the error in about.rst as well! The PR is good to merge now @Q-lds. I'll provide my approval

**Review by vishal-ph — APPROVED** [review]
Duplicates fixed and about.rst message reverted

**Pull request merged**